src/ai/backend/manager/actions/monitors/prometheus.py

```diff
@@ -0,0 +1,25 @@
+from typing import override
+
+from ai.backend.common.metrics.metric import ActionMetricObserver
+from ai.backend.manager.actions.action import BaseAction, BaseActionTriggerMeta, ProcessResult
+from ai.backend.manager.actions.monitors.monitor import ActionMonitor
+
+
+class PrometheusMonitor(ActionMonitor):
```

Comment:
I think this should be `PrometheusReporter`, not Monitor

Reply:
The reporter only wants to apply tasks that are okay to be done in other layers, like message queue,
and the monitor wants to attach tasks that can be attached directly, so the Monitor implementation makes sense here.